### PR TITLE
aurora improvements

### PR DESCRIPTION
**Pull request by TonDraaijer**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -138,11 +153,35 @@ variable "min_capacity" {
...
+variable "performance_insights" {
+  description = "Specifies whether Performance Insights is enabled or not."
+  type        = bool
+  default     = false
+}
+
+variable "performance_insights_kms_key_id" {
+  description = "The ARN for the KMS key to encrypt Performance Insights data."
+  type        = string
+  default     = ""
```
Comment (marwinbaumannsbp) [inline variables.tf:176]:
```suggestion
  default     = null
```

File: variables.tf [hunk 2]
```diff
@@ -138,11 +153,35 @@ variable "min_capacity" {
...
+  default     = null
+}
+
 variable "password" {
   type        = string
   description = "Password for the master DB user"
 }
 
+variable "performance_insights" {
+  description = "Specifies whether Performance Insights is enabled or not."
```
Comment (marwinbaumannsbp) [inline variables.tf:168]:
```suggestion
  description = "Specifies whether Performance Insights is enabled or not"
```

File: variables.tf [hunk 3]
```diff
@@ -138,11 +153,35 @@ variable "min_capacity" {
...
+}
+
+variable "performance_insights_kms_key_id" {
+  description = "The ARN for the KMS key to encrypt Performance Insights data."
+  type        = string
+  default     = ""
+}
+
+variable "permissions_boundary" {
+  description = "The ARN of the policy that is used to set the permissions boundary for the role."
```
Comment (marwinbaumannsbp) [inline variables.tf:180]:
```suggestion
  description = "The ARN of the policy that is used to set the permissions boundary for the role"
```

File: variables.tf [hunk 4]
```diff
@@ -138,11 +153,35 @@ variable "min_capacity" {
...
   description = "Password for the master DB user"
 }
 
+variable "performance_insights" {
+  description = "Specifies whether Performance Insights is enabled or not."
+  type        = bool
+  default     = false
+}
+
+variable "performance_insights_kms_key_id" {
```
Comment (marwinbaumannsbp) [inline variables.tf:173]: We can remove this variable and reuse `kms_key_id`

**Review by marwinbaumannsbp — CHANGES_REQUESTED** [review]

**Review by marwinbaumannsbp — DISMISSED** [review]

**Review by svanharmelen — CHANGES_REQUESTED** [review]
As for the variables, please keep a consistent ordering (type, default, description)... Will have a closer look at the rest later today.

**Comment by banovicn** [comment]
let me rearrange (sort by name) all the variables and double check ordering to fulfil needed format

**Comment by svanharmelen** [comment]
Just FYI for anyone following this PR... We discussed it and it should be fine now. But before we merge it @banovicn will ask to test the updated code one more. If all is good, we can merge the PR and tag a new release.

**Comment by marwinbaumannsbp** [comment]
> Just FYI for anyone following this PR... We discussed it and it should be fine now. But before we merge it @banovicn will ask to test the updated code one more. If all is good, we can merge the PR and tag a new release.

@svanharmelen the module has been tested successfully, can you merge and tag a new release?

**Review by svanharmelen — APPROVED** [review]

**Comment by svanharmelen** [comment]
Done... Released version v0.4.0.

**Pull request merged**